src/pages/data_submission/DataAccessGovernance.jsx

```diff
@@ -221,61 +220,45 @@ export const DataAccessGovernance = (props) => {
     setAllConsentGroupsSaved(consentGroupsState.every((state) => state.editMode === false));
   }, [consentGroupsState, setAllConsentGroupsSaved]);
 
-  return div({
-    className: 'data-submitter-section',
-  }, [
-    h2('Data Access Governance'),
-    div({},
-      [
-        // consent groups
-        consentGroupsState
-          ?.map((state, idx) => {
-            if (isNil(state)) {
-              return div({}, []);
-            }
-
-            return div({ key: state.key },
-              [
-                h(ConsentGroupForm, {
-                  idx: idx,
-                  dacs: dacs,
-                  consentGroupsState,
-                  studyEditMode,
-                  disableDelete: consentGroupsState.length === 1,
-                  saveConsentGroup: (newGroup) => updateConsentGroup(idx, newGroup.value, newGroup.valid),
-                  deleteConsentGroup: () => deleteConsentGroup(idx),
-                  updateNihInstitutionalCertificationFile: (file) => updateNihInstitutionalCertificationFile(idx, file),
-                  startEditConsentGroup: () => startEditConsentGroup(idx),
-                  validation: validation?.consentGroups?.at(idx) || {},
-                  onValidationChange: (change) => {
-                    onValidationChange({ ...change, ...{ key: `consentGroups[${idx}].` + change.key } });
-                  },
-                  datasetNames: datasetNames
-                })
-              ]
-            );
-          }),
-
-        // add consent group
-        div({
-          className: 'right-header-section',
-          id: 'add-new-consent-group-btn',
-          style: {
-            display: 'flex',
-            alignItems: 'flex-end',
-            margin: '2rem 0 2rem 0'
+  return (
+    <div className='data-submitter-section'>
+      <h2>Data Access Governance</h2>
+      <div>
+        {consentGroupsState?.map((state, idx) => {
+          if (isNil(state)) {
+            return null;
```

Comment:
```suggestion
            return <div></div>;
```

I think we might want to return an empty div instead to keep it equivalent to the previous code, but this seems to work too.